index.html

```diff
@@ -135,6 +135,71 @@ <h2 class="cover-title">
   </div>
 </div>
 
+<section class="latest-news">
+  <header>
+    <a class="latest-news-title" href="#">Latest news</a>
+  </header>
+  <div class="latest-article-container">
+    <article class="latest-article">
+      <img class="latest-article-photo" src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/california-car-ev-hydrogen-future-electruc-fcev-ev-480x270.png" alt="image" width="75" height="75">
```

Comment:
Each article has an image and a title (`hx` tag), both of them with a link (`a` tag)